plugin/trino-base-jdbc/src/main/java/io/trino/plugin/jdbc/DefaultJdbcMetadata.java

```diff
@@ -113,11 +113,17 @@ public class DefaultJdbcMetadata
 
     private final AtomicReference<Runnable> rollbackAction = new AtomicReference<>();
 
-    public DefaultJdbcMetadata(JdbcClient jdbcClient, boolean precalculateStatisticsForPushdown, Set<JdbcQueryEventListener> jdbcQueryEventListeners)
+    private final SyntheticColumnHandleBuilder syntheticColumnBuilder;
+
+    public DefaultJdbcMetadata(JdbcClient jdbcClient,
+            boolean precalculateStatisticsForPushdown,
```

Comment:
We format arguments like below when putting arguments on separate lines:
```java
    public DefaultJdbcMetadata(
            JdbcClient jdbcClient,
            boolean precalculateStatisticsForPushdown,
            Set<JdbcQueryEventListener> jdbcQueryEventListeners,
            SyntheticColumnHandleBuilder syntheticColumnBuilder)
```

Same for Ignite and Phoenix.